src/pages/explore/dex/_components/ExploreDTokensGrid.tsx

```diff
@@ -0,0 +1,130 @@
+                price={value.denominationPrice}
+              />
+            </div>
+          ))}
+      </div>
+
+      <div className="md:hidden block w-full mt-8">
+        <Slider className="" {...sliderSettings}>
+          <div className="flex flex-col space-y-5">
+            {selectionTokens &&
+              Object.entries(selectionTokens)
+                .slice(0, 3)
+                .map(([, value], key) => (
+                  <ExploreDTokenCard
+                    key={key}
+                    id={`mobile-${value.token.symbol}`}
+                    symbol={value.token.displaySymbol}
+                    name={value.token.name}
+                    price={value.denominationPrice}
+                  />
+                ))}
+          </div>
+          <div className="flex flex-col space-y-5">
+            {selectionTokens &&
+              Object.entries(selectionTokens)
+                .slice(3, 6)
+                .map(([, value], key) => (
+                  <ExploreDTokenCard
+                    key={key}
+                    id={`mobile-${value.token.symbol}`}
+                    symbol={value.token.displaySymbol}
+                    name={value.token.name}
+                    price={value.denominationPrice}
+                  />
+                ))}
+          </div>
+          <div className="flex flex-col space-y-5">
+            {selectionTokens &&
+              Object.entries(selectionTokens)
+                .slice(6)
+                .map(([, value], key) => (
+                  <ExploreDTokenCard
+                    key={key}
+                    id={`mobile-${value.token.symbol}`}
+                    symbol={value.token.displaySymbol}
+                    name={value.token.name}
+                    price={value.denominationPrice}
+                  />
+                ))}
+          </div>
+        </Slider>
+      </div>
+    </>
+  );
+}
+
+function ExploreDTokenCard({
+  symbol,
+  name,
+  price,
+  id,
+}: {
+  symbol: string;
+  name: string;
+  price: string;
+  id: string;
+}) {
+  const Icon = getAssetIcon(symbol.replace("d", ""), true, id);
+  // eslint-disable-next-line prefer-const
+  let { suffix, value } = useUnitSuffix(price, 2);
+  if (!suffix) {
+    value = new BigNumber(value).toFixed(2);
+  }
+  return (
+    <div className="card-outline-2 p-[0.5px] rounded-[15px]">
+      <div className="rounded-[15px] bg-dark-00 py-6 px-[31.2px] w-full">
+        <div className="flex md:flex-col flex-row md:gap-x-0 gap-x-4">
+          <Icon className="h-12 w-12 place-self-center md:mb-4" />
+          <div className="flex flex-col">
+            <div className="font-bold text-xl md:text-center">{symbol}</div>
+            <div className="text-xs text-dark-800 md:text-center line-clamp-1">
+              {name}
+            </div>
+            <div className="mt-3 lg:text-[18px] lg:leading-6 text-sm text-dark-1000 font-semibold md:text-center">
+              ${value}
```

Comment:
since we are not re-using `value`, we can remove line 106, 108-110, and use `const` at line 107
```suggestion
              ${new BigNumber(value).toFixed(2)}
```

Reply:
updated !